content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -54,7 +56,7 @@ To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/
    * kubelet:
       * `--feature-gates="IPv6DualStack=true"`
    * kube-proxy:
-      * `--proxy-mode=ipvs`
+      * `--proxy-mode=ipvs` or `--proxy-mode=iptables` (since 1.18)
```

Comment:
dumb question because I don't know how the docs are published, do we need to mention ` (since 1.18)` ?
or these docs are only published with the 1.18 version?

Reply:
only 1.18 support iptables + dual stack.
1.17 supports ipvs + dual stack only.